engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-docker-repositories-list.controller.js

```diff
@@ -22,16 +22,24 @@ angular.module('Bastion.content-views').controller('ContentViewDockerRepositorie
             'organization_id': CurrentOrganization,
             'content_view_id': $scope.$stateParams.contentViewId,
             'content_type': 'docker'
-
         },
         'queryUnpaged');
 
-        nutupane.masterOnly = true;
```

Comment:
You'll definitely want this in there or you will get errors about not defining `closeTable()`.